akka-typed-tests/src/test/scala/akka/typed/persistence/scaladsl/PersistentActorCompileOnlyTest.scala

```diff
@@ -401,17 +403,17 @@ object PersistentActorCompileOnlyTest {
     PersistentActor.immutable[Command, Event, State](
       persistenceId = "myPersistenceId",
       initialState = (),
-      actions = Actions { (_, cmd, _) ⇒
+      commandHandler = CommandHandler { (_, _, cmd) ⇒
         cmd match {
           case Enough ⇒
-            Persist(Done)
-              .andThen(
-                SideEffect(_ ⇒ println("yay")),
-                Stop())
+
+            Effect.persist(Done)
+              .andThen((_: State) ⇒ println("yay"))
```

Comment:
it's not required to write the type right? just showing it off here?

Reply:
Let me double check that

Reply:
Dang. It is needed, or explicitly typing `Effect.persist` the type inference does not work when there are two `andThen` chained for some reason, but it does work when there is only one. Hmmm.

Reply:
Actually, if I use the parameter less variant it works fine